fundamentals/fundamentals-bio/src/main/scala/com/github/pshirshov/izumi/functional/bio/package.scala

```diff
@@ -1,12 +1,157 @@
-package com.github.pshirshov.izumi.functional
+package com.github.pshirshov.izumi.functional {
 
-import com.github.pshirshov.izumi.functional.mono.SyncSafe
+  import com.github.pshirshov.izumi.functional.mono.SyncSafe
 
-package object bio {
-  type SyncSafe2[F[_, _]] = SyncSafe[F[Nothing, ?]]
 
-  object SyncSafe2 {
-    def apply[F[_, _] : SyncSafe2]: SyncSafe2[F] = implicitly
+  package object bio extends B1 {
```

Comment:
Can't say I follow the trick here :/

Reply:
As I understand, the idea is to move implicits conversions for BIO to package object since, but I miss the point of why we have to use type aliases and lazy vals. What's reason behind the SyncSafe2, etc.

Another concern, since package object will be deprecated in Scala 3.0, will this approach be backward compatible or not?